lib/mpl_toolkits/axes_grid1/mpl_axes.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, axes):
         def __getitem__(self, k):
             if isinstance(k, tuple):
                 r = SimpleChainedObjects(
-                    [super(Axes.AxisDict, self).__getitem__(k1) for k1 in k])
+                    [super().__getitem__(k1) for k1 in k])
```

Comment:
This one will fail.  The scope of a list comprehension in Py3 is not what you might expect.  #10527 tripped over this, and had to change it back.